inc/taskjobview.class.php

```diff
@@ -411,13 +414,56 @@ public function ajaxModuleItemsDropdown($options) {
             break;
 
       }
+
+      // filter actor list with active agent and with current module active
+      $condition = "";
+      if ($moduletype == "actors"
+          && in_array($itemtype, ["Computer", "PluginFusioninventoryAgent"])) {
+         // remove install suffix from deploy
+         $modulename = str_replace('DEPLOYINSTALL', 'DEPLOY', strtoupper($method));
```

Comment:
DEPLOYINSTALL no longer exist because it has been updated with migration script in 9.1+1.0 version